src/api/notion.ts

```diff
@@ -56,7 +56,7 @@ const queryCollectionBody = {
   query: { aggregations: [{ property: "title", aggregator: "count" }] },
   loader: {
     type: "table",
-    limit: 999,
+    limit: 100,
```

Comment:
This is not true for the `queryCollectionBody`, we had 300+ more entries in a table for translations and I could set this limit back to 999